libraries/core_libs/consensus/src/transaction/transaction_manager.cpp

```diff
@@ -114,11 +115,11 @@ std::pair<bool, std::string> TransactionManager::insertTransaction(const std::sh
     }
   }
 
-  if (const auto [is_valid, reason] = verifyTransaction(trx); !is_valid) {
+  if (const auto [status, reason] = verifyTransaction(trx); status != TransactionStatus::Verified) {
     return {false, reason};
   }
 
-  if (insertValidatedTransactions({trx})) {
+  if (insertValidatedTransactions({{trx, TransactionStatus::Verified}})) {
```

Comment:
why is this hardcoded to TransactionStatus::Verified ?

Reply:
It is the only possible value but it is better to actually use the result of verification. Fixed